src/components/BookCard/BookCard.jsx

```diff
@@ -1,22 +1,22 @@
 import React from "react";
 import "./BookCard.css";
 
-
 export default function BookCard({
   imageLinks = {},
   authors = [],
   title,
   publishedDate,
 }) {
-  return (
-    <div className="card day-theme">
-      <img src={`../images/${imageLinks.thumbnail}`} alt="Book cover" />
-      <div className="card-content">
-        <p className="card-title"> {title}</p>
-        <p className="card-author">
-          By {authors.join(",")} ({publishedDate})
-        </p>
-      </div>
-    </div>
-  );
+
+	return (
+		<div className="card">
+			<img src={`../images/${imageLinks.thumbnail}`} alt="Book cover" />
+			<div className="card-content">
+				<p className="card-title"> {title}</p>
+				<p className="book-author">
+					By {authors.join(', ')} {publishDate}
```

Comment:
When I run this code, it crashes saying that `publishDate` on this line is undefined.

Reply:
yes this i already changed in another branch. there is supposed to be publishedDate apart of publishDate.